src/middlewared/middlewared/schema/string_schema.py

```diff
@@ -34,7 +34,7 @@ class Str(EnumMixin, Attribute):
 
     def __init__(self, *args, **kwargs):
         # Sqlite limits ( (2 ** 31) - 1 ) for storing text - https://www.sqlite.org/limits.html
-        self.max_length = kwargs.pop('max_length', 1024) or (2 ** 31) - 1
+        self.max_length = kwargs.pop('max_length', 4096) or (2 ** 31) - 1
```

Comment:
Why are we increasing this?

Reply:
A motd longer than 1024 characters can't be saved as it gives us a validation error, DoD needs 1300 minimum so I upped it for that. Shouldn't change the size stored in the DB, sqlite3 makes all string/varchars/etc into Text which as a limit of some really large number.

Reply:
We should pass `max_length` to the motd field, not change the default value for all fields.

Reply:
You can simply set `max_length=None` to remove the max length limitation in the Str schema for motd.

Reply:
Didn't realize that was an option lol, fixing that right now